From ca4cc6f009db97e1f75cd1b49f0541d3f47dbd1c Mon Sep 17 00:00:00 2001
From: me-cafebabe <me_cafebabe@hotmail.com>
Date: Wed, 7 Sep 2022 07:54:19 +0000
Subject: [PATCH 3/6] core: Export TARGET_INIT_VENDOR_LIB to Soong

---
 core/soong_config.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/soong_config.mk b/core/soong_config.mk
index 16b7faecf..ec4ed7c31 100644
--- a/core/soong_config.mk
+++ b/core/soong_config.mk
@@ -143,6 +143,8 @@ $(call add_json_bool, Malloc_zero_contents,              $(call invert_bool,$(fi
 $(call add_json_bool, Malloc_pattern_fill_contents,      $(MALLOC_PATTERN_FILL_CONTENTS))
 $(call add_json_str,  Override_rs_driver,                $(OVERRIDE_RS_DRIVER))
 
+$(call add_json_str,  Target_init_vendor_lib,            $(TARGET_INIT_VENDOR_LIB))
+
 $(call add_json_bool, UncompressPrivAppDex,              $(call invert_bool,$(filter true,$(DONT_UNCOMPRESS_PRIV_APPS_DEXS))))
 $(call add_json_list, ModulesLoadedByPrivilegedModules,  $(PRODUCT_LOADED_BY_PRIVILEGED_MODULES))
 
-- 
2.30.2

