From 603067d1763ce5a9dd52533008f3b69451f532bc Mon Sep 17 00:00:00 2001
From: Build Server <build@localhost>
Date: Fri, 8 Jul 2022 09:28:10 +0000
Subject: [PATCH] set SKIP_ABI_CHECKS=true

---
 build/envsetup.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/build/envsetup.sh b/build/envsetup.sh
index c324db30..432e3369 100644
--- a/build/envsetup.sh
+++ b/build/envsetup.sh
@@ -947,3 +947,5 @@ function fixup_common_out_dir() {
         mkdir -p ${common_out_dir}
     fi
 }
+
+export SKIP_ABI_CHECKS=true
-- 
2.30.2

