From 91e9f4e6823411abeca59c34b6ad21467792ab73 Mon Sep 17 00:00:00 2001
From: Android Build Server <build@AndroidBuild>
Date: Thu, 5 Oct 2023 07:20:23 +0000
Subject: [PATCH] Drop deprecated clang property

---
 ipacm/Android.bp  | 1 -
 ipanat/Android.bp | 2 --
 2 files changed, 3 deletions(-)

diff --git a/ipacm/Android.bp b/ipacm/Android.bp
index a925c5d..88221e6 100644
--- a/ipacm/Android.bp
+++ b/ipacm/Android.bp
@@ -38,7 +38,6 @@ cc_binary {
     ],
 
     init_rc: ["src/ipacm.rc"],
-    clang: true,
     vendor: true,
 
     shared_libs: [
diff --git a/ipanat/Android.bp b/ipanat/Android.bp
index 310304d..7a8ecb7 100644
--- a/ipanat/Android.bp
+++ b/ipanat/Android.bp
@@ -23,6 +23,4 @@ cc_library_shared {
         "-Wall",
         "-Werror",
     ] + ["-DFEATURE_IPA_ANDROID"],
-
-    clang: true,
 }
-- 
2.30.2

