From c61492c26ca74a7f3f3917c6407a12244b3bd8df Mon Sep 17 00:00:00 2001
From: Android Build Server <build@AndroidBuild>
Date: Thu, 5 Oct 2023 07:42:43 +0000
Subject: [PATCH 1/2] Drop deprecated vendor_service attr

---
 common/dynamic/service.te | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/dynamic/service.te b/common/dynamic/service.te
index 1016a18..7453853 100644
--- a/common/dynamic/service.te
+++ b/common/dynamic/service.te
@@ -1 +1 @@
-type hal_lineage_health_service, vendor_service, hal_service_type, service_manager_type;
+type hal_lineage_health_service, hal_service_type, service_manager_type;
-- 
2.30.2

