From 0437ef32682164669d2aa0da7d03e2f8b396a5ae Mon Sep 17 00:00:00 2001
From: me-cafebabe <87921983+me-cafebabe@users.noreply.github.com>
Date: Sun, 2 Jan 2022 16:41:35 +0000
Subject: [PATCH] hal: msm8916: Allow to override mixer file path with prop

---
 hal/msm8916/platform.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/hal/msm8916/platform.c b/hal/msm8916/platform.c
index 5ab79799d..3f456a3bf 100644
--- a/hal/msm8916/platform.c
+++ b/hal/msm8916/platform.c
@@ -1515,6 +1515,13 @@ static void query_platform(const char *snd_card_name,
             sizeof(msm_device_to_be_id_internal_codec) / sizeof(msm_device_to_be_id_internal_codec[0]);
 
     }
+
+#if 1
+    char mixer_xml_path_custom[PROPERTY_VALUE_MAX];
+    property_get("vendor.audio.mixer_xml.path", mixer_xml_path_custom, "");
+    if (strlen(mixer_xml_path_custom) > 0)
+        strlcpy(mixer_xml_path, mixer_xml_path_custom, sizeof(mixer_xml_path_custom));
+#endif
 }
 
 void platform_set_echo_reference(struct audio_device *adev, bool enable,
-- 
2.30.2

