From 476b44e9e4e79214ae9cb0b1bb13952779d3a64a Mon Sep 17 00:00:00 2001
From: Build Server <build@localhost>
Date: Thu, 30 Jun 2022 11:21:47 +0000
Subject: [PATCH 25/26] Enable console by default

---
 rootdir/init.rc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 5cab603b8..defc2c7b9 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -1032,10 +1032,10 @@ service ueventd /system/bin/ueventd
 service console /system/bin/sh
     class core
     console
-    disabled
+#    disabled
 #   user shell
     group shell log readproc
-    seclabel u:r:shell:s0
+    seclabel u:r:su:s0
     setenv HOSTNAME console
 
 on property:ro.debuggable=1
-- 
2.30.2

