From d4d8aca6faebfb9cdda6900cd2522ff8674ad441 Mon Sep 17 00:00:00 2001
From: Build Server <build@localhost>
Date: Fri, 8 Jul 2022 09:28:10 +0000
Subject: [PATCH] set SKIP_ABI_CHECKS=true

---
 build/envsetup.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/build/envsetup.sh b/build/envsetup.sh
index 2e057090..099d616b 100755
--- a/build/envsetup.sh
+++ b/build/envsetup.sh
@@ -902,3 +902,5 @@ export TEMPORARY_DISABLE_PATH_RESTRICTIONS=true
 
 # Define our ROM for SmartBuild-enabled device trees
 export SMARTBUILD_RELEASE=revengeos
+
+export SKIP_ABI_CHECKS=true
-- 
2.30.2

