From 15bd5fdd7599efe3be52f81c83b23b29f9bff532 Mon Sep 17 00:00:00 2001
From: me-cafebabe <me_cafebabe@hotmail.com>
Date: Wed, 7 Sep 2022 12:44:34 +0000
Subject: [PATCH 2/2] init: Use product_variables for specifying vendor init
 lib

---
 init/Android.bp | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/init/Android.bp b/init/Android.bp
index 3df0e917c..a2d0b3c27 100644
--- a/init/Android.bp
+++ b/init/Android.bp
@@ -216,7 +216,6 @@ cc_library_static {
     defaults: [
         "init_defaults",
         "selinux_policy_version",
-        "vendor_init_defaults",
     ],
     srcs: init_common_sources + init_device_sources,
     generated_sources: [
@@ -233,6 +232,12 @@ cc_library_static {
         export_proto_headers: true,
     },
 
+    product_variables: {
+        target_init_vendor_lib: {
+            whole_static_libs: ["%s"],
+        },
+    },
+
     target: {
         recovery: {
             cflags: ["-DRECOVERY"],
-- 
2.30.2

