From b8392a9c17742bb3a4800514d1cdffe782f8e3aa Mon Sep 17 00:00:00 2001
From: Android Build Server <build@AndroidBuild>
Date: Tue, 31 Jan 2023 06:40:03 +0000
Subject: [PATCH] fix gapps makefile path

---
 config/versioning.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/versioning.mk b/config/versioning.mk
index f5b4dd4..022d422 100644
--- a/config/versioning.mk
+++ b/config/versioning.mk
@@ -25,7 +25,7 @@ MIKU_ROM_VERSION := TDA_0.9.0
 TARGET_PRODUCT_SHORT := $(subst miku_,,$(TARGET_DEVICE))
 
 ifeq ($(MIKU_GAPPS), true)
-    $(call inherit-product, vendor/gms/gms.mk)
+    $(call inherit-product, vendor/gapps/common/common-vendor.mk)
     MIKU_TYPE = GAPPS-
 endif
 
-- 
2.30.2

