From 2126b415d0684e2a7573671c7caebd97db6ffb39 Mon Sep 17 00:00:00 2001
From: Build Server <build@localhost>
Date: Fri, 8 Jul 2022 11:17:28 +0000
Subject: [PATCH] fix x86_64 build

---
 Android.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Android.mk b/Android.mk
index 566b02e..411b0b2 100644
--- a/Android.mk
+++ b/Android.mk
@@ -31,6 +31,9 @@ LOCAL_REQUIRED_MODULES := \
 
 LOCAL_MODULE_TARGET_ARCH := arm arm64 x86 x86_64
 my_src_arch := $(call get-prebuilt-src-arch,$(LOCAL_MODULE_TARGET_ARCH))
+ifeq ($(my_src_arch),x86_64)
+my_src_arch := x86
+endif
 LOCAL_SRC_FILES := prebuilt/$(my_src_arch)/webview.apk
 
 LOCAL_PREBUILT_JNI_LIBS_arm := @lib/armeabi-v7a/libwebviewchromium.so
-- 
2.30.2

