From 8a14fb036bbdfd150fdfcfe6bc16c94478f3dbe3 Mon Sep 17 00:00:00 2001
From: me-cafebabe <me_cafebabe@hotmail.com>
Date: Wed, 7 Sep 2022 12:44:34 +0000
Subject: [PATCH 2/2] init: Use product_variables for specifying vendor init
 lib

---
 init/Android.bp | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/init/Android.bp b/init/Android.bp
index 6d7d9ba3e..a5e5d18b8 100644
--- a/init/Android.bp
+++ b/init/Android.bp
@@ -223,7 +223,6 @@ cc_defaults {
     defaults: [
         "init_defaults",
         "selinux_policy_version",
-        "vendor_init_defaults",
     ],
     srcs: init_common_sources + init_device_sources,
     export_include_dirs: ["."],
@@ -240,6 +239,12 @@ cc_defaults {
         export_proto_headers: true,
     },
 
+    product_variables: {
+        target_init_vendor_lib: {
+            whole_static_libs: ["%s"],
+        },
+    },
+
     target: {
         recovery: {
             cflags: ["-DRECOVERY"],
-- 
2.39.2

